### PR TITLE
Adding survey form

### PR DESCRIPTION
**Pull request by nejidevelops**
On this PR:
- I added my css and html files
- I worked with HTML form tag to create my survey form

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Microverse Trials
+# Microverse Trials - -Survey Form
 
 ## Albert Antwi
 ## Newton Ombese
```
Comment (nejidevelops) [inline README.md:4]: - Nice efforts there in the README, you've identified the collaborators of the project :+1: , but I would recommend you to add a more detailed README which gives more insights of the project even before someone checks it out. 

You can use this [README template](https://github.com/microverseinc/readme-template). It is well descriptive but kindly customize it to your project. :grin:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                </label>
+            </div>
+            <!-- <div class="form-group">
+                <p>Which IT field do you love working on?</p>
+                <select name="mostLike" id="most-like" class="form-control" required>
+                  <option disabled selected value>Select an option</option>
+                  <option value="webdeveloper">Frontend</option>
+                  <option value="backend">Backend</option>
+                </select>
+            </div> -->
```
Comment (nejidevelops) [inline index.html:60]: While comments can be useful in explaining **complex logic or documentations**, it is essential to have **clean code** and consider the type of comment you have in your code. I think the above comment is _cluttering your code_ and _making it dirty_. I would like to recommend you to clean it up and make sure your code looks clean and professional :sunglasses::business_suit_levitating:

File: style.css [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+  border-radius: 0.5rem;
+  opacity: 0.8;
+}
+
+.survey-form {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
```
Comment (nejidevelops) [inline style.css:110]: - [OPTIONAL] I had the opportunity to go through both your HTML and CSS codes and I have identified that there is nowhere in your HTML that you've used the class name **survey-form**. This change may be optional but it plays a big role in identifying your professionalism, be sure to remove it :wastebasket: and make your code :sparkles: clean.

**Review by nejidevelops — COMMENTED** [review]
Hi @NewtonOmbese, 

Good job so far! I love the project and the color selections :+1: :grin: 

There are some issues that you still need to work on to get your project to the next level, once you resolve them you'll be good to go :wink: 

Alt Text
## Highlights
Well-structured files ✅
Semantic practice ✅
All linter checks are passing ✅

## Required Changes ♻️

- [ ] Kindly note that it is important to write a **descriptive pull request summary** because it helps other developers and team members to understand the changes that have been made. 
    - Additionally, writing a descriptive pull request summary help code reviewer know exactly the changes you have made, and that way they can focus on just those particular changes made. Therefore, taking the time to write a descriptive pull request summary can improve the efficiency and effectiveness of the code review process, and ultimately help ensure the overall quality of the code.

- [ ] Kindly check at the suggested changes and make them to take your coding skills to the next level :sunglasses: I believe you are a top-tier developer :100: 

_Check the comments under the review._

Cheers and Happy coding!👏👏👏 :man_technologist:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cabraham1 — APPROVED** [review]
Hi @NewtonOmbese , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good Inline Comment✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**